### DISCUSSION TITLE
Is there any way to hide the modal's drag indicator?

### DISCUSSION THREAD

**Discussion by willianferreiradev** [gorhom/react-native-bottom-sheet#1201] [General]
I searched and couldn't find a way to do this.

**Comment by willianferreiradev** [comment]
@gorhom please

**Reply by agestaun** [reply]
Are you referring to this?
<img width="200" alt="Screenshot 2023-01-05 at 15 29 21" src="https://user-images.githubusercontent.com/9607291/210803660-4d7a5d05-e6e9-4346-a35d-8913af1a874f.png">

If that's the case, you have many ways to do it using styles.

`handleIndicatorStyle={{ display: none }}`

`handleIndicatorStyle={{ width: 0 }}`

`handleIndicatorStyle={{ height: 0 }}`

etc...

It would be nice to have a prop to hide it though. Or to pass null.
